tests/wpt/meta/notifications/permissions-non-secure.html.ini

```diff
@@ -1,10 +1,10 @@
 [permissions-non-secure.html]
   expected: ERROR
   [Notification.requestPermission must be called from a secure context]
```

Comment:
These passing subtest results can be removed from this file. Both the [..] and expected lines are now unnecessary.